src/server/hotspot_partition_calculator.cpp

```diff
@@ -98,5 +103,49 @@ void hotspot_partition_calculator::data_analyse()
     }
 }
 
+/*static*/ void hotspot_partition_calculator::server_hotkey_detect_send(
+    const std::string &app_name,
+    const int partition_index,
+    const hotkey_detect_type write_hotkey_detect,
+    const hotkey_detect_action stop_detect_hotkey)
```

Comment:
```suggestion
    const hotkey_detect_action action)
```

Reply:
`stop_detect_hotkey` is clearer to understand this variable I think